### PR TITLE
Update docs for ActiveSpeakerObserver.md

### PR DESCRIPTION
**Pull request by SteveMcFarlin**

### CODE REVIEW

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+{: #ActiveSpeakerObserver}
+
+<section markdown="1">
+
+> `@inherits` [RtpObserver](#RtpObserver)
+
+An active speaker observer monitors the speech activity of the selected audio producers. It just handles audio producers (if [addProducer()](#rtpObserver-addProducer) is called with a video producer it will fail).
+
+Audio levels used for speech detection are read from an RTP header extension. No decoding of audio data is done. See [RFC6464](https://tools.ietf.org/html/rfc6464) for more information.
+
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:11]: Can we mention here that this class uses the algorithm of Jitsi with a link to the spec?

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+### Events
+{: #ActiveSpeakerObserver-events}
+
+<section markdown="1">
+
+See also [RtpObserver Events](#RtpObserver-events).
+
+#### ActiveSpeakerObserver.on("dominantspeaker", fn(event))
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:74]:
```suggestion
#### ActiveSpeakerObserver.on("dominantspeaker", fn(dominantSpeaker))
```

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+{: #ActiveSpeakerObserver-on-dominantspeaker .code}
+
+Emitted when a new dominant speaker is detected.
+
+<div markdown="1" class="table-wrapper L3">
+
+Argument  | Type    | Description   
+--------- | ------- | ----------------
+`dominantspeaker` | property&lt;[Producer](#Producer)&gt; | Producer Object
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:83]: What's that "property"?

```suggestion
`dominantSpeaker` | [Producer](#Producer) | Audio producer with most dominant audio level in the last interval
```

File: _includes/documentation/v3/mediasoup/api/Router.md [hunk 4]
```diff
@@ -285,6 +285,30 @@ const transport2 = await router2.createWebRtcTransport({ ... });
 const consumer2 = await transport2.consume({ producerId: producer1.id, ... });
 ```
 
+#### router.createActiveSpeakerObserver(options)
+{: #router-createActiveSpeakerObserver .code}
+
+Creates a new audio level observer.
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/Router.md:291]:
```suggestion
Creates a new active speaker observer.
```

File: _includes/documentation/v3/mediasoup/api/Router.md [hunk 5]
```diff
@@ -285,6 +285,30 @@ const transport2 = await router2.createWebRtcTransport({ ... });
...
+
+<div markdown="1" class="table-wrapper L3">
+
+Argument    | Type    | Description | Required | Default
+----------- | ------- | ----------- | -------- | ----------
+`options`   | [ActiveSpeakerObserverOptions](#ActiveSpeakerObserverOptions) | Options. | Yes |
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/Router.md:297]: Are those options really required? no default values?

File: _includes/documentation/v3/mediasoup/api/Router.md [hunk 6]
```diff
@@ -285,6 +285,30 @@ const transport2 = await router2.createWebRtcTransport({ ... });
...
+
+> `@async`
+>
+> `@returns` [ActiveSpakerObserver](#ActiveSpeakerObserver)
+
+```javascript
+const activeSpeakerObserver = await router.createActiveSpeakerObserver(
+	{
+		interval   : 500
+	});
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/Router.md:309]:
```suggestion
const activeSpeakerObserver = await router.createActiveSpeakerObserver(
  {
    interval   : 500
  });
```

File: _includes/documentation/v3/mediasoup/api/Router.md (hunk 5 above)
Comment (ibc) [reply _includes/documentation/v3/mediasoup/api/Router.md:297]: (it's ok if required, just asking)

**Review by ibc — CHANGES_REQUESTED** [review]
Thanks. Just a few changes requested.

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md (hunk 1 above)
Comment (SteveMcFarlin) [reply _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:11]: For sure. I will mention it similar to how it is in the ActiveSpeakerObserver header file.

Comment (ibc) [reply _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:11]: Great.

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md (hunk 3 above)
Comment (SteveMcFarlin) [reply _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:83]: I will make this change. The event is not fired for every interval. Perhaps I should make that clear. The interval setting for the class is only used for how often it is tested. An event will only be emitted when there is a dominant speaker change.

File: _includes/documentation/v3/mediasoup/api/Router.md (hunk 5 above)
Comment (SteveMcFarlin) [reply _includes/documentation/v3/mediasoup/api/Router.md:297]: No. I believe they are optional, but I need to make sure.

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md (hunk 3 above)
Comment (SteveMcFarlin) [reply _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:83]: btw. I am not sure where the property came from. Initially I was editing the file with a markdown editor, then switch to doing it with a regular text editor.

File: _includes/documentation/v3/mediasoup/api/Router.md [hunk 7]
```diff
@@ -294,7 +294,7 @@ Creates a new active speaker observer.
 
 Argument    | Type    | Description | Required | Default
 ----------- | ------- | ----------- | -------- | ----------
-`options`   | [ActiveSpeakerObserverOptions](#ActiveSpeakerObserverOptions) | Options. | Yes |
+`options`   | [ActiveSpeakerObserverOptions](#ActiveSpeakerObserverOptions) | Options. | No |
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/Router.md:297]: Good, I've also fix it in `AudioLevelObserver`: https://github.com/versatica/mediasoup-website/commit/8b558b485501a778f932a4f74216f1015ccbf950

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md [hunk 8]
```diff
@@ -0,0 +1,102 @@
...
+
+### Events
+{: #ActiveSpeakerObserver-events}
+
+<section markdown="1">
+
+See also [RtpObserver Events](#RtpObserver-events).
+
+#### ActiveSpeakerObserver.on("dominantspeaker", fn(dominantSpeaker))
+{: #ActiveSpeakerObserver-on-dominantspeaker .code}
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:75]:
```suggestion
#### activeSpeakerObserver.on("dominantspeaker", fn(dominantSpeaker))
{: #activeSpeakerObserver-on-dominantspeaker .code}
```

File: _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md [hunk 9]
```diff
@@ -0,0 +1,102 @@
...
+{: #ActiveSpeakerObserver-observer-events}
+
+<section markdown="1">
+
+See also [RTP Observer Observer Events](#RtpObserver-observer-events).
+
+#### ActiveSpeakerObserver.observer.on("dominantspeaker", fn(event))
+{: #ActiveSpeakerObserver-observer-on-dominantspeaker .code}
+
+Same as the [dominantspeaker](#ActiveSpeakerObserver-on-dominantspeaker) event.
```
Comment (ibc) [inline _includes/documentation/v3/mediasoup/api/ActiveSpeakerObserver.md:100]:
```suggestion
#### activeSpeakerObserver.observer.on("dominantspeaker", fn(event))
{: #activeSpeakerObserver-observer-on-dominantspeaker .code}

Same as the [dominantspeaker](#activeSpeakerObserver-on-dominantspeaker) event.
```

**Review by ibc — CHANGES_REQUESTED** [review]
Just two changes requested and we are done :)

**Comment by SteveMcFarlin** [comment]
This PR should be done! Thanks for the review.

**Review by ibc — APPROVED** [review]

**Pull request merged**